config/routes.rb

```diff
@@ -11,6 +11,8 @@
 
   post '/cart', to: 'cart#create'
 
+  delete '/cart', to: 'cart#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.